lib/cartopy/crs.py

```diff
@@ -2117,6 +2117,45 @@ def y_limits(self):
         return self._y_limits
 
 
+class Aitoff(_WarpedRectangularProjection):
+    """
+    An Aitoff projection.
+
+    This projection is a modified azimuthal equidistant projection, balancing
+    shape and scale distortion. There are no standard lines and only the
+    central point is free of distortion.
+
+    """
+
+    _handles_ellipses = False
+
+    def __init__(self, central_longitude=0, false_easting=None,
+                 false_northing=None, globe=None):
+        """
+        Parameters
+        ----------
+        central_longitude: float, optional
+            The central longitude. Defaults to 0.
+        false_easting: float, optional
+            X offset from planar origin in metres. Defaults to 0.
+        false_northing: float, optional
+            Y offset from planar origin in metres. Defaults to 0.
+        globe: :class:`cartopy.crs.Globe`, optional
+            If omitted, a default globe is created.
+
+            .. note::
+                This projection does not handle elliptical globes.
```

Comment:
It doesn't look like this note appears anywhere in the docs? It seems like all of the __init__ docstrings are left out currently.

Reply:
Hmm, maybe it was with the change in doc themes? I don't remember what the older one looked like now.

Reply:
We can open up another issue if we want to update the docs later.